src/compiler/gen.js

```diff
@@ -604,14 +604,14 @@ ${type}.${key} = ${f};
   return res;
 }
 // Generates the overarching parent type for an enum.
-function generateParent(type, parts, static = {}) {
+function generateParent(type, parts, static_item = {}) {
   let res = (
     `let ${type} = {
   order: ${zStringify(parts)},
   ${parts.join(",\n\t")}
 };`
   );
-  res += generateStatics(type, static);
+  res += generateStatics(type, static_item);
```

Comment:
had to make this change as the name conflicted with the modifier

Reply:
yeah - that's fine!